atlas/config.py

```diff
@@ -69,4 +70,25 @@
     # https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
     urllib3.disable_warnings()
 
+# List of statuses to do inactive checks on.
```

Comment:
Let's move these variables to `config_local.py` since this would most likely be customized on a per user or per environment basis.